Lib/test/test_enum.py

```diff
@@ -1776,7 +1776,8 @@ def _missing_(cls, item):
                     # trigger not found
                     return None
         self.assertIs(Color('three'), Color.blue)
-        self.assertRaises(ValueError, Color, 7)
+        with self.assertRaises(ValueError):
```

Comment:
This statement passes on the old code as well. Is it possible to specifically assert that a `ValueError` has been raised, but without the second `ValueError` being raised in the process?